plugin.xml

```diff
@@ -59,7 +59,7 @@
         <config-file target="AndroidManifest.xml" parent="application">
           <provider
               android:name="org.apache.cordova.camera.FileProvider"
-              android:authorities="${applicationId}.provider"
+              android:authorities="${applicationId}.camera.provider"
```

Comment:
If these can conflict with other plugins I think I'd prefer to stay away from generic terms such as `camera`, otherwise I feel like there is still a decent risk of running into this kind of problem in the future.

Therefore I think something like `"${applicationId}.cordova.plugin.camera.provider"` will be better, as it provides a pretty specific context.

```suggestion
              android:authorities="${applicationId}.cordova.plugin.camera.provider"
```